### PR TITLE
Feature/add phrases

### PR DESCRIPTION
**Pull request by Canario0**

### CODE REVIEW

File: src/routes/(auth)/+page.svelte [hunk 1]
```diff
@@ -50,11 +52,12 @@
       removeable={phrase.user_id === user_id}
       liked={phrase.likes.some((like) => like.user_id === user_id)}
       likeCount={phrase.likes.length}
-      on:cancel={() => deletePhrase(phrase.id)}
-      on:like={() => likedPhrase(phrase.id)}
-      on:unlike={() => unLikePhrase(phrase.id)}
+      on:cancel={async () => await deletePhrase(phrase.id)}
+      on:like={async () => await likedPhrase(phrase.id)}
+      on:unlike={async () => await unLikePhrase(phrase.id)}
```
Comment (ismtabo) [inline src/routes/(auth)/+page.svelte]: **chore:** it doesn't have much sense defining async function only with an await

Doesn't this have the same effect if rewrite as follows?
```suggestion
      on:cancel={async () => deletePhrase(phrase.id)}
      on:like={async () => likedPhrase(phrase.id)}
      on:unlike={async () => unLikePhrase(phrase.id)}
```

File: src/routes/(auth)/api/v1/phrases/+server.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ export const POST: RequestHandler = async (event) => {
   if (session == null) {
     throw error(403);
   }
-  const data = await request.formData();
-  const content = data.get("content")?.toString();
+  const data = await request.json();
```
Comment (ismtabo) [inline src/routes/(auth)/api/v1/phrases/+server.ts:18]: **chore:** ensure body types

Just ensure body type is right or create an issue with the todo.

**Review by ismtabo — CHANGES_REQUESTED** [review]

**Comment by ismtabo** [comment]
**issue:** Where does the add phrase component appear?

image

File: src/routes/(auth)/+page.svelte (hunk 1 above)
Comment (Canario0) [reply src/routes/(auth)/+page.svelte]: Yes

**Review by ismtabo — APPROVED** [review]

**Pull request merged**